src/py/flwr/client/supernode/app.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower SuperNode."""
+
+import argparse
+from logging import DEBUG, INFO
+
+from flwr.common import EventType, event
+from flwr.common.exit_handlers import register_exit_handlers
+from flwr.common.logger import log
+
+
+def run_supernode() -> None:
+    """Run Flower SuperNode."""
+    log(INFO, "Starting Flower SuperNode")
+
+    event(EventType.RUN_SUPERNODE_ENTER)
+
+    args = _parse_args_run_supernode().parse_args()
+
+    log(
+        DEBUG,
+        "Flower will load ClientApp `%s`",
+        getattr(args, "client-app"),
+    )
```

Comment:
@danieljanes, how does this relate to the multiapp support?

Reply:
It doesn't. Once we merge the implementation of this command, the main way to load apps will be from the app install dir. We might want to keep support for this arg though to make dev/prototyping easier.